docs/milestones.md

```diff
@@ -0,0 +1,19 @@
+# Milestones
+
+### [M0] Milestone 0: Modelo del problema
+
++ **Descripción**: Analizar las historias de usuario para modelar los elementos fundamentales del dominio. 
```

Comment:
En general, cuando una persona solicita revisión poco tiempo después de una evaluación en la cual hay errores conceptuales graves, es porque tratar de entender cuál es el problema y actuar en consecuencia no es algo que entre en sus cálculos. En consecuencia, a veces también se entiende mal o superficilamente lo que se dice en el comentario y se actúa de peor forma todavía.
En la corrección en esta línea te dije:
> ¿El producto es "analizar"? ¿Eso es un producto o una tarea?

Como consecuencia, has borrado "producto". ¿Has entendido lo que es un milestone? ¿Crees que borrar "producto " refleja mejor el concepto de milestone que tú tienes? Entiendo que es así, puesto que es lo que has hecho.

Adicionalmente, has cambiado "el sistema básico para asignación de turnos" por "elementos fundamentales del dominio". Al menos has borrado "asignación de turnos"... pero como lo has hecho a toda prisa y por lo que he dicho en el comentario, *no has aplicado lo mismo en el siguiente milestone*. 

En todo caso, ¿"elementos fundamentales del dominio" qué son? Si alguien te indica que ese es el PMV que quiere, ¿qué le vas a entregar? 

Si no entiendes lo que es un modelo de problema, trata de entenderlo. No se trata de que combinéis palabras y frases hasta que déis con la tecla, sino que entendáis qué pasos hay que dar para desarrollar un producto y la importancia que tiene cada uno de los pasos.

Por si fuera poco, has cambiado una historia de usuario concreta por "las historias de usuario". ¿Has leído los errores frecuentes?